docs/source/dev_guide/project-standards.rst

```diff
@@ -0,0 +1,190 @@
+Project Standards
+=================
+
+Development Environment
+-----------------------
+If you haven't already, please visit :ref:`"(b) Development Environment" <dev-env>`
+to set up your local development environment.
+
+Version Control (VC)
+--------------------
+
+The repository uses a fork-based Git workflow with tag releases.
+
+.. figure:: git-flow.svg
+   :alt: Git Flow Diagram
+
+Guidelines for VC
+~~~~~~~~~~~~~~~~~
+1. ``master`` must always be **deployable**
+2. All changes are made through **support** branches on forks
+3. **Rebase** with ``master`` to avoid/resolve conflicts
+4. Make sure ``pre-commit`` checks pass when committing (enforced in CI/CD build)
+5. Open a pull-request (PR) early for discussion
+6. Once the CI/CD build passes and PR is approved, **squash and rebase** your
+   commits
+7. Merge PR into ``master`` and **delete the branch**
+
+Things to Avoid in VC
+~~~~~~~~~~~~~~~~~~~~~
+1. Don't merge in broken or commented out code
+2. Don't commit onto ``master`` directly
+3. Don't merge with conflicts (handle conflicts upon rebasing)
+
+Source: https://gist.github.com/jbenet/ee6c9ac48068889b0912
+
+Pre-commit
+~~~~~~~~~~
+The repository uses the ``pre-commit`` package to manage pre-commit hooks.
+These hooks help enforce QA standards and identify simple issues
+at the commit level before submitting code reviews.
+
+.. figure:: pre-commit-flow.svg
+   :alt: Pre-commit Flow Diagram
+
+   ``pre-commit`` Flow
+
+Helpful ``pre-commit`` Commands
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Install into your cloned repo ::
+
+    conda activate zstash_dev
+    pre-commit install
+
+Automatically run all pre-commit hooks (just commit) ::
+
+    # Tip: If there is an issue with pre-commit, you can bypass with the `--no-verify` flag. Please do NOT use this on a regular basis.
+    git commit -m '...'
+
+.. figure:: ../_static/pre-commit-passing.png
+   :alt: pre-commit Output
+
+   ``pre-commit`` Output
+
+Manually run all pre-commit hooks ::
+
+    pre-commit run --all-files
+
+Run individual hook ::
+
+    # Available hook ids: trailing-whitespace, end-of-file-fixer, check-yaml, black, isort, flake8, mypy
+    pre-commit run <hook_id>
+
+Squash and Rebase Commits
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Before you merge a support branch back into ``master``, the branch is typically
+squashed down to a single* buildable commit, and then rebased on top of the main repo's ``master`` branch.
+
+\* *In some cases, it might be logical to have multiple squashed commits, as long as each commit passes the CI/CD build*
+
+Why squash and rebase commits?
+
+- Ensures build passes from the commit
+- Cleans up Git history for easy navigation
+- Makes collaboration and review process more efficient
+- Makes handling conflicts from rebasing simple since you only have to deal with conflicted commits
```

Comment:
`Makes handling conflicts from rebasing simple since there are fewer commits to fix conflicts on`